frontend/src/store/facilityBooking/action.ts

```diff
@@ -179,13 +134,12 @@ export const editBookingDescription = (newBookingDescription: string) => (dispat
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const setViewDates = (newDate: any, selectedFacilityId: number) => (dispatch: Dispatch<ActionTypes>) => {
+export const setViewDates = (newDate: any) => (dispatch: Dispatch<ActionTypes>) => {
   const startDate = newDate
   const endDate = newDate
```

Comment:
is creating consts necessary?

Reply:
Will need it next time when we implement weekly view thus didn't remove it. It was part of the old code.